src/test/groovy/edu/ie3/datamodel/io/csv/CsvFileDefinitionTest.groovy

```diff
@@ -116,8 +119,8 @@ class CsvFileDefinitionTest extends Specification {
     actual == expected
 
     where:
-    manipulatedDirectory                 || expected
-    ""                                   || this.fileName
-    FilenameUtils.concat("test", "grid") || FilenameUtils.concat(FilenameUtils.concat("test", "grid"), this.fileName)
+    manipulatedDirectory    || expected
+    Path.of("")             || Path.of(this.fileName)
+    Path.of("test", "grid") || Paths.get("test", "grid", this.fileName)
```

Comment:
Could be `Paths.of(...)` for the expected part as well, you'd also need one import less then